parent-poms/pom.xml

```diff
@@ -816,7 +840,7 @@
                                         <exclude>org.codehaus.mojo:versions-maven-plugin:2.2</exclude>
                                     </excludes>
                                 </bannedDependencies>
-                                <dependencyConvergence/>
```

Comment:
How come this was left out, it's actually a very difficult to debug source of error  sometimes.